quick-install.md

```diff
@@ -0,0 +1,26 @@
+# Install Thunder
+
+### System requirements
+
+You have to install `PHP`, `composer` and `git` on your computer before you can install Thunder. The `composer` requires `git` command for proper functioning.
+
+To install `PHP` please take a look at official [Installation and Configuration](https://www.php.net/manual/install.php) for `PHP`.
+On top of `PHP`, you need to install required libraries. Thunder installation requires the following `PHP` libraries: `xmlwriter`, `gd`, `mbstring`, `curl`, `json` and `pdo_sqlite`. The installation of `PHP` extensions can defer between operating systems, that's why you should check for detailed instructions on `PHP` [Installation and Configuration](https://www.php.net/manual/install.php) page.
+
+To install `composer`, you can check [installation instructions](https://getcomposer.org/download) and for `git` you can find installation instructions [here](https://git-scm.com/downloads).
```

Comment:
you can check the 'composer' [installation...

Reply:
👍